<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanCreator</class>
 <widget class="QWidget" name="PlanCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tworzenie planu</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>65</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Nazwa:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>65</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Opis:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Wyświetlanie stanu:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>231</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="descriptionEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>231</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>65</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Zadania:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>490</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>zatwierdź</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="openAccess">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Otwarty dostęp              </string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="stateModeCombo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>231</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>tekstowy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>odhaczanie</string>
    </property>
   </item>
  </widget>
  <widget class="QPlainTextEdit" name="statesEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Podaj stany po przecinku</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>501</width>
     <height>421</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>499</width>
      <height>419</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="taskList"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
